src/uu/ls/src/ls.rs

```diff
@@ -437,14 +440,42 @@ impl Config {
             IndicatorStyle::None
         };
 
+        let mut ignore_patterns = GlobSetBuilder::new();
+        if options.is_present(options::IGNORE_BACKUPS) {
+            ignore_patterns.add(Glob::new("*~").unwrap());
+            ignore_patterns.add(Glob::new(".*~").unwrap());
+        }
+
+        for pattern in options.values_of(options::IGNORE).into_iter().flatten() {
+            match Glob::new(pattern) {
+                Ok(p) => {
+                    ignore_patterns.add(p);
+                }
+                Err(_) => show_warning!("Invalid pattern for ignore: '{}'", pattern),
+            }
+        }
+
+        if files == Files::Normal {
+            for pattern in options.values_of(options::HIDE).into_iter().flatten() {
+                match Glob::new(pattern) {
```

Comment:
would it be possible to add a test to cover this case?

Reply:
Interesting, that `match` is definitely tested, however, I think the second arm was not covered so I added a test for that. Hopefully that makes Codecov happy :)

Reply:
Hmm, the new CI has run and it's still there. I really have no idea what codecov means here, because it's definitely covered...